src/fastertransformer/kernels/quantization_tensor.h

```diff
@@ -57,23 +57,23 @@ void invokePerColDequantizationInt8(T*            dst,
                                     cudaStream_t  stream = 0);
 
 template<typename T>
-void invokePerColQuantizationInt4x2(int8_t*       dst,
-                                    const T*      src,
+void invokePerColQuantizationInt4x2(const T*      src,
```

Comment:
Is this kernel only used for rocm? If so, please seperate it into a dedicated kernel file for rocm. if not, you can not simply change this but remain the cuda version unchanged.

Reply:
Understand. 
Recovered this files to commit 8c993e73b6b8d3aa29d0d548ca92e5e76e60394e
And move all rocm related code into `quantization_rocm.h/cu`